lib/resources/users.js

```diff
@@ -113,7 +113,8 @@ module.exports = (service, endpoint) => {
     auth.actor.map((actor) =>
       ((queryOptions.extended === true)
         ? Promise.all([ Users.getByActorId(actor.id).then(getOrNotFound), Auth.verbsOn(actor.id, '*') ])
-          .then(([ user, verbs ]) => Object.assign({ verbs }, user.forApi()))
+          .then(([ user, verbs ]) => Users.getPreferences(user.actorId)
+            .then((preferences) => Object.assign(Object.assign({ verbs }, user.forApi()), preferences)))
```

Comment:
Can you do `Object.assign({ verbs, preferences }, user.forApi())))` instead instead of the 2 nested object assigns?

Reply:
I agree that it'd be nice to avoid the nested `Object.assign()`. `Object.assign()` can take more than two arguments, so you could also do:

```js
Object.assign({ verbs }, user.forApi(), preferences)
```